README.md

```diff
@@ -48,6 +48,51 @@ For sling's management UI, you can head to http://localhost:8181/starter/index.h
 - To list all bundles and view their status: bundle:list
 - Find out why a bundle is in waiting state: diag _[bundle-id]_
 
+### Run the application using Podman
+First, install [podman](https://podman.io).
+
+Then, create a pod:
+
+```sh
+podman pod create --name pantheon-karaf -p 8181 -p 5005
+```
+
+This will create a `pantheon-karaf` pod with ports 8181 (for web access) and 5005 (for
+remote Java debugging) open.
+
+Run a mongo database container in the pod.
+
+```sh
+podman run --pod pantheon-karaf --name slingmongo -d mongo
+```
+
+Build the pantheon docker image
+
+```sh
+buildah bud --layers -f container/Dockerfile -t pantheon-karaf-app .
```

Comment:
I am  not sure, if it's an issue at my end, but this step is throwing error message on my local.

Error: Transaction failed
error building at STEP "RUN yum -y install java-1.8.0-openjdk": error while running runtime: exit status 1
ERRO exit status 1

Reply:
It works fine at my end. I used to receive that error if the network was wonky.